.github/workflows/build-release.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         platform:
           - 'linux-x64'
           - 'linuxmusl-x64'
+          - 'linuxmusl-arm64v8'
```

Comment:
Note that this target currently assumes it's running on an ARM64 device, which GitHub Actions doesn't support. AFAIK, only  [Travis supports Linux ARM64](https://docs.travis-ci.com/user/multi-cpu-architectures/) in an early beta stage, only for OSS-projects.

Perhaps we could cross-compile for `aarch64-unknown-linux-musl` on `x86_64-unknown-linux-musl` instead?

Reply:
That is why I suggested using buildx, seems easy to do, like this sample:
https://github.com/n1ru4l/node-sqlite3/actions/runs/177214307/workflow

Reply:
https://github.com/docker/setup-buildx-action appears to use QEMU for ARM - I would prefer the cross-compilation approach suggested by Kleis.